source/documentation/deploying-an-app/container-repositories/requirements.html.md.erb

```diff
@@ -41,3 +42,15 @@ USER 1001
 ```
 
 If you need help to run your container as a non-root user, please speak to your team or consult the image vendor's documentation.
+
+## Setting UTC timezone inside your container
+
+Our cluster from the node level down down to the metrics collection is based on the Coordinated Universal Time (UTC). We consciously avoid local timezones to avoid any [issues and bugs around daylight savings](http://yellerapp.com/posts/2015-01-12-the-worst-server-setup-you-can-make.html). Most containers are UTC by default but some are not. For debian based containers the following steps will set the TZ to UTC:
```

Comment:
```suggestion
Our cluster from the node level  down to the metrics collection is based on the Coordinated Universal Time (UTC). We consciously avoid local timezones to avoid any [issues and bugs around daylight savings](http://yellerapp.com/posts/2015-01-12-the-worst-server-setup-you-can-make.html). Most containers are UTC by default but some are not. For debian based containers the following steps will set the TZ to UTC:
```